lib/jekyll-mentions.rb

```diff
@@ -97,8 +99,8 @@ def filter_regex
           Regexp.new(
             HTML::Pipeline::MentionFilter::MentionPatterns[mention_username_pattern]
           )
-        rescue TypeError
-          %r!@\w+!
+                          rescue TypeError
+                            %r!@\w+!
         end
```

Comment:
this doesn't make sense since the end should be in line with `rescue`?

Reply:
better now?